### PR TITLE
User type addition

### PR DESCRIPTION
**Pull request by keithtkj**
Collaborators: Keith, Menelik, Alexander
Added business schema and business routes account creation

### CODE REVIEW

File: backend/routes/business_route.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import express from 'express'
+import auth from '../controllers/auth_controller.js'
```
Comment (parkaransingh) [inline backend/routes/business_route.js:2]: Need to test these routes in future sprint with business accounts.

**Review by parkaransingh — APPROVED** [review]
Will test locally first.

File: backend/models/business_model.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import mongoose from 'mongoose'
```
Comment (madhura112) [inline backend/models/business_model.js:1]: Good code structure/organization

**Review by madhura112 — APPROVED** [review]
Looks great

**Pull request merged**